ci/build-qemu-powernv.sh

```diff
@@ -17,24 +17,25 @@ branch="master"
 head_sha="$(git ls-remote --heads $repo $branch | cut -f 1)"
 cachedir="ci_cache/qemu/$head_sha/"
 
-if [ -f "$cachedir/qemu-system-ppc64" ] ; then
-	mkdir -p qemu/ppc64-softmmu/
-	cp $cachedir/qemu-system-ppc64 qemu/ppc64-softmmu/
-	exit 0
-fi
+#if [ -f "$cachedir/qemu-system-ppc64" ] ; then
+#	mkdir -p qemu/build/
+#	cp -p $cachedir/qemu-system-ppc64 ./qemu/build/.
+#	exit 0
+#fi
```

Comment:
Why are you disabling the build cache? The whole point of it was to reduce the cycle time for CI jobs since rebuilding qemu for every job is largely pointless.

Also, commenting things out randomly and just leaving them lying around isn't acceptable. Commenting things out without any kind of explanation especially so.

Reply:
Yes, it is very much required for multiple runs on which it will help to save time. The qemu-system-binary is copied with all the required permission. However still while executing the test it says qemu command not found. Need further debug to fix it. Hence commented for time now. so that the pull request should not be stopped due this known issue.